cyences_app_for_splunk/default/data/ui/views/cs_windows_patch.xml

```diff
@@ -0,0 +1,122 @@
+<form theme="dark" version="1.1">
+  <init>
+    <set token="tkn_patch_numbers">*</set>
+  </init>
+  <label>Windows Patch</label>
+  <search id="data_check">
+    <query>| tstats count where `cs_wineventlog_system` host="$host$" by host 
```

Comment:
Let's not add $host$ in the base search. This will not have no performance advantage.

```
| tstats count where `cs_wineventlog_system` by host 
| eval data="Data Present" 
| inputlookup all_windows_hosts.csv append=true
| stats first(data) as data by host
......
```

Add filter in the search that is using the base search.